winton_kafka_streams/processor/_stream_thread.py

```diff
@@ -89,43 +87,55 @@ def __init__(self, _topology, _config, _kafka_supplier):
 
         self.topics = _topology.topics
 
-        log.info('Topics for consumer are: %s', self.topics)
+        self.thread = threading.Thread(target=self.run)
+        self.log = logging.getLogger(__name__ + '(' + self.thread.name + ')')
```

Comment:
Thoughts on the line above? It's not awful, and does what it should, but all the self.log slightly annoys me.